lib/src/http_headers_impl.dart

```diff
@@ -14,7 +14,7 @@ import 'http_exception.dart';
 
 class HttpHeadersImpl implements HttpHeaders {
   final Map<String, List<String>> headers;
-  final String protocolVersion;
+  final String _protocolVersion;
```

Comment:
There's no problem with implementing a read-only property w/ a final field...right?

Reply:
That's a good point. I'll remove the explicit getter and just expose the field in HttpHeaders.